config/manager/manager.yaml

```diff
@@ -26,8 +26,6 @@ spec:
     spec:
       securityContext:
         runAsNonRoot: true
-        seccompProfile:
-          type: RuntimeDefault
```

Comment:
Why are we removing this?

Reply:
Because it is not needed and in fact it is blocking the CVP tests to pass